plugins/scaffolder/src/next/TemplateWizardPage/Stepper/createAsyncValidators.ts

```diff
@@ -42,7 +42,7 @@ export const createAsyncValidators = (
         if (validator) {
           const fieldValidation = createFieldValidation();
           try {
-            await validator(value, fieldValidation, context);
+            await validator(value, fieldValidation, { ...context, formData });
```

Comment:
In terms of documentation, could you add a use case example [here](https://github.com/backstage/backstage/blob/master/docs/features/software-templates/writing-custom-field-extensions.md) in the official docs as well as [TSDoc @example](https://tsdoc.org/pages/tags/example/) to update the [API reference page](https://backstage.io/docs/reference/plugin-scaffolder.customfieldvalidator)?

There's a note discouraging relying on formData [here](https://backstage.io/docs/features/software-templates/writing-custom-field-extensions#access-data-from-other-fields). That consideration seems to be valid for validators as well, so we could mention same note in these new examples as well.

Reply:
Sure, will add those